.eslintrc.js

```diff
@@ -9,12 +9,18 @@ module.exports = {
     'plugin:ember-suave/recommended'
   ],
   env: {
-    browser: true
+    browser: true,
+    es6: true
   },
   rules: {
     'arrow-parens': ['error', 'as-needed'],
     // Destructuring is not supported in node 4.8.0
     'ember-suave/prefer-destructuring': 'off',
-    'operator-linebreak': ['error', 'before', { 'overrides': { '=': 'ignore'} } ]
+    'operator-linebreak': ['error', 'before', { 'overrides': { '=': 'ignore'} } ],
```

Comment:
Do we need these custom rules here?

Reply:
Need is a strong word. New versions of eslint prefer a strange indentation style that I believe is objectively less readable. The rules introduced here allow the current styling to pass linting checks without change.

Reply:
So something in the eslint:recommended set changed?

Reply:
Exactly. Indentation for function args was borked.